src/widget/wknob.cpp

```diff
@@ -57,6 +57,16 @@ void WKnob::guiTick() {
 }
 
 void WKnob::inputActivity() {
+    // Bug #1793015: Previously we would simply call QWidget::update in response
+    // to input events that required re-rendering widgets, relying on Qt to
+    // batch them together and deliver them at a reasonable frequency. On macOS,
+    // the behavior of QWidget::update seems to have changed such that render
+    // events happen much more frequently than they used to. To address this, we
```

Comment:
It may be helpful when looking back on this in the future to explain this change happened when switching from Qt4 to Qt5.

Reply:
Oops, Qt 5 was in my original draft but got dropped. Re-added